src/Raven.Server/Web/System/TcpConnectionInfoHandler.cs

```diff
@@ -19,5 +27,116 @@ public Task Get()
 
             return Task.CompletedTask;
         }
+
+        [RavenAction("/info/tcp/pull", "GET", AuthorizationStatus.UnauthenticatedClients)]
```

Comment:
Let's use a better name here: `info/remote-task/topology`, for example
This won't be the only usage of the endpoint